ambry-router/src/main/java/com/github/ambry/router/PutOperation.java

```diff
@@ -485,7 +484,8 @@ void maybeNotifyForBlobCreation() {
             RouterUtils.getAccountContainer(accountService, getBlobProperties().getAccountId(),
                 getBlobProperties().getContainerId());
         NotificationBlobType blobType = isComposite() ? NotificationBlobType.Composite
-            : options.isChunkUpload() ? NotificationBlobType.DataChunk : NotificationBlobType.Simple;
+            : options.isChunkUpload() || options.skipCompositeChunk() ? NotificationBlobType.DataChunk
```

Comment:
In some sense, 
options.isChunkUpload is "ambry native chunk upload"
options.skipCompositeChunk is "s3 chunk upload".

Probably in next PR, we can do some naming change for it to consolidate them.

Reply:
Sure.